### DISCUSSION TITLE
STAR 2.7.7a release: STARconsensus

### DISCUSSION THREAD

**Discussion by alexdobin** [alexdobin/STAR#1103] [General]
Dear STAR Users,

STAR 2.7.7a has been released:
https://github.com/alexdobin/STAR/releases/tag/2.7.7a

It introduces STARconsensus, which allows mapping RNA-seq to the consensus genome, which is built by inserting major alleles into the reference genome.
Benjamin Kaminow @kaminow led the efforts to assess the increase of mapping accuracy with consensus genomes.
For more details, please check out bioRxiv preprint:
https://www.biorxiv.org/content/10.1101/2020.12.22.423111v1
Benjamin also developed a separate tool for constructing consensus VCF from public databases:
https://github.com/kaminow/ConsDB

As always, questions, comments, and suggestions are very welcome!

**Comment by maciej-jonczyk** [comment]
Dear Dr. Dobin,

I've read your article https://doi.org/10.1101/gr.275613.121 with great interest.

I plan to use STAR-consensus approach to construct pangenome for maize for RNA-seq analysis for 4 maize inbred lines.
I have found appropriate collection of data in VCF files (resequencing of various, ca 1500 inbred lines, Schanble's dataset from https://wgs.maizegdb.org/). This data contain VCF for inbred lines used by me.
I wonder, how many lines should I include in pangenome analysis. I wonder if I could use almost all 1500, excluding maize relatives like Tripsacum and Zea  mexicana.
I have access to server with 24 cores and 128 Gb RAM and wonder in this setup allow such analysis.

I have also second question, I haven't found exact number of individuals you have used to construct human pangenome. Could you, please provide this number?
